lib/iron_bank/local.rb

```diff
@@ -51,17 +51,11 @@ def load_records
     def csv_options
       {
         headers:           true,
-        header_converters: header_converters,
+        header_converters: [->(h) { IronBank::Utils.underscore(h).to_sym }],
```

Comment:
If `headers` will still fit within the 80 chars, we can remove the reek exception.

Reply:
`header` doesn't fit the 80 chars 😞